label_studio/core/settings/base.py

```diff
@@ -739,3 +739,7 @@ def collect_versions_dummy(**kwargs):
 
 CLOUD_STORAGE_CHECK_FOR_RECORDS_PAGE_SIZE = get_env('CLOUD_STORAGE_CHECK_FOR_RECORDS_PAGE_SIZE', 10000)
 CLOUD_STORAGE_CHECK_FOR_RECORDS_TIMEOUT = get_env('CLOUD_STORAGE_CHECK_FOR_RECORDS_TIMEOUT', 60)
+
+CONTEXTLOG_SYNC = False
```

Comment:
Do we want to `get_bool_env` for this one too? Or is it never settable through env variables? How would this be set then?

Reply:
This variable shouldn't really be settable, we specify it here in the settings to that we can override it via the settings fixture in conftest.